docs/src/pages/index.tsx

```diff
@@ -6,41 +6,274 @@ import clsx from "clsx";
 // There should be a line here that says
 // import React from "react";
 // VSCode might try to delete it, but that will break the site.
-import React from "react";
+import React, { useState } from "react";
 import styles from "./index.module.css";
+import '../css/style.css'
 
 function HomepageHeader() {
   const { siteConfig } = useDocusaurusContext();
+  const [isActive, setIsActive] = useState(false);
+  const toggleMenu = () => {
+    setIsActive((prevState) => !prevState);
+  };
+
+
   return (
-    <header className={clsx("hero hero--primary", styles.heroBanner)}>
-      <div className="container">
-        <h1 className="hero__title">{siteConfig.title}</h1>
-        <p className="hero__subtitle">{siteConfig.tagline}</p>
-        <div className={styles.buttons}>
-          <Link
-            className="button button--secondary button--lg"
-            to="/docs/getting-started"
-          >
-            Tutorial - ⏱️ 5 min
-          </Link>
+      <header className="header">
+        <div className="container">
+          <div className="header-wrapper">
+            <a href="" className="logo"><img src="img/dev_logo_dark.svg" alt="logo"/></a>
+            <ul className={`menu ${isActive ? "active" : ""}`}>
+              <li className="contain-logo"><a href="" className="logo"><img src="img/logo.png" alt=""/></a></li>
+              <li className="menu-item"><a className="menu-link" href="https://code.groundlight.ai/python-sdk/docs/getting-started">Docs</a></li>
+              <li className="menu-item"><a className="menu-link" href="https://code.groundlight.ai/python-sdk/docs/building-applications">Applications</a></li>
+              <li className="menu-item"><a className="menu-link" href="https://code.groundlight.ai/python-sdk/api-reference-docs/">API Reference</a></li>
+              <li className="menu-item"><a className="menu-link" href="https://github.com/groundlight/python-sdk">GitHub</a></li>
+              <li className="menu-item"><a href="https://dashboard.groundlight.ai/" className="cmn-button outline">Login</a></li>
+            </ul>
+            <button onClick={toggleMenu} className={`menu-toggle ${isActive ? "active" : ""}`}>
+              <img className="toggle" src="img/burger-menu-right-svgrepo-com.svg" alt="menu toggle"/>
+              <img className="closee" src="img/close-svgrepo-com.svg" alt="menu close"/>
```

Comment:
nit: typo "closee"

Reply:
perhaps this is intended?

Reply:
> perhaps this is intended?

yes, it's intended, because `close` itself was working another way. so I changed it to `closee`